spring-cloud-gcp-starters/spring-cloud-gcp-starter-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/autoconfig/GcpPubSubAutoConfiguration.java

```diff
@@ -86,16 +88,25 @@ public ChannelProvider subscriberChannelProvider() {
 	@ConditionalOnMissingBean(name = "publisherChannelProvider")
 	public ChannelProvider publisherChannelProvider() {
 		return TopicAdminSettings
-				.defaultChannelProviderBuilder()
+				.defaultGrpcChannelProviderBuilder()
 				.setClientLibHeader(DEFAULT_SOURCE_NAME,
 						this.getClass().getPackage().getImplementationVersion())
 				.build();
 	}
 
 	@Bean
+	@ConditionalOnMissingBean(name = "pubsubMessageConverter")
+	public MessageConverter pubsubMessageConverter() {
```

Comment:
I think the proper bean name has to be `pubSubMessageConverter`

Reply:
thnx! actually this was meant for Joao's ref; wasn't expecting this to be taken at the moment :D